testing/src/uploadResultTQ.js

```diff
@@ -0,0 +1,58 @@
+var request = require('request');
+var fs = require('fs');
+import * as dotenv from "dotenv";
+
+
+
+token = undefined
+const tq_username = process.env.TQ_USER_NAME;
+const tq_password = process.env.TQ_PASSWORD;
+const tq_secret = process.env.TQ_SECRET;
+var options = {
+  'method': 'POST',
+  'url': 'https://api.testquality.com/api/oauth/access_token',
+  'headers': {
+        'Content-Type': 'application/json'
+  },
+  body: JSON.stringify({
+    "grant_type": "password",
+    "client_id": "2",
+    "client_secret": `"${TQ_SECRET}"`,
+    "username": `"${TQ_USER_NAME}"`,
+    "password": `"${TQ_PASSWORD}"`
+  })
+};
+function loginToTQ (callback){
+    request(options, function (error, response) {
+        if (error) throw new Error(error);
+        const obj =  JSON.parse(response.body)
+        token = obj["access_token"]
+        // console.log("Fetching the TQ Token")
+        callback()
```

Comment:
It would be good to use async/await pattern instead callback to be aligned with the rest of the solution.